tests/cypress/e2e/issues_prs2/issue_8952_interpolation_impossible.js

```diff
@@ -0,0 +1,179 @@
+            secondX: shape.secondX + delta,
+        };
+    }
+    if (axis === 'y') {
+        return {
+            ...shape,
+            firstY: shape.firstY + delta,
+            secondY: shape.secondY + delta,
+        };
+    }
+    return null;
+}
+
+function shapeToPayload(shape, frame, shapeType) {
+    return {
+        frame,
+        type: shapeType,
+        points: [shape.firstX, shape.firstY, shape.secondX, shape.secondY],
+    };
+}
+
+function makeTrack(shapePayloads, frame0, trackLabel) {
+    return {
+        shapes: shapePayloads,
+        frame: frame0,
+        labelName: trackLabel,
+        objectType: 'track',
+    };
+}
+
+context('Create any track, check if track works correctly after deleting some frames', () => {
+    const precision = 0.01; // db server precision is 2 digits
+    const stash = {};
+    function storeShape(num) {
+        cy.get('.cvat_canvas_shape').invoke('attr', 'x').then((x) => {
+            cy.get('.cvat_canvas_shape').invoke('attr', 'y').then((y) => {
+                stash[num] = { x: parseFloat(x), y: parseFloat(y) };
+            });
+        });
+    }
+    function compareShape(num) {
+        const { x, y } = stash[num];
+        cy.get('.cvat_canvas_shape').invoke('attr', 'x').then((xVal) => {
+            expect(parseFloat(xVal)).to.be.closeTo(x, precision);
+        });
+        cy.get('.cvat_canvas_shape').invoke('attr', 'y').then((yVal) => {
+            expect(parseFloat(yVal)).to.be.closeTo(y, precision);
+        });
+    }
+    describe('Description: user error, Could not receive frame 43 No one left position or right position was found. Interpolation impossible', () => {
+        let jobID = null;
+        const delta = 300;
+        before(() => {
+            cy.visit('/auth/login');
+            cy.login();
+
+            // Create assets for task using nodeJS
+            cy.imageGenerator(imagesFolder, imageFileName, width, height, color, posX, posY, labelName, imagesCount);
+            cy.createZipArchive(dirToArchive, archivePath);
+            cy.createTaskFromArchive(taskName, labelName, archiveName);
+
+            cy.goToTaskList();
+            cy.openTaskJob(taskName);
+            cy.url().should('contain', 'jobs').then((url) => {
+                const last = url.lastIndexOf('/');
+                jobID = parseInt(url.slice(last + 1), 10);
+            }).then(() => {
+                // Remove all annotations and draw a track rect
+                const wrap = (shape, frame) => shapeToPayload(shape, frame, 'rectangle');
+                const shape0 = createRectangleShape2Points;
+                const shape1 = translateShape(shape0, delta, 'x');
+                const shape2 = translateShape(shape1, delta, 'y'); // TODO: fix coords, rect flies away
+                const track = makeTrack([
+                    wrap(shape0, 0),
+                    wrap(shape1, 2),
+                    wrap(shape2, 4),
+                ], 0, labelName);
```

Comment:
Please, just change initial object structure as `headlessCreateObjects` expects. Making this thing dynamic is not obvious and looks too complex.